### DISCUSSION TITLE
Upload/Download limit by tag

### DISCUSSION THREAD

**Discussion by SethFalco** [jesec/flood#815] [Ideas]
Would it be possible to make it so that Flood has upload/download limits by tag?

For my use-case, I have torrents from a variety of sources:

* Linux ISOs like Debian, Ubuntu, and Tuxedo OS
* Humble Bundle games and soundtracks
* Web Archive
* etc

And have them tagged accordingly. However, some of them I want to seed, while for others I don't.

**Comment by trim21** [comment]
you mean total rate limit by tag? or each torrent rate limit by tag?

**Reply by SethFalco** [reply]
Hmm, I think for my use case specifically, the difference is minimal. So, if this is implemented, you can decide whichever you think makes sense. I'm not sure which would be more desirable for users tbh. 🤔 

For clarity my use-case is to make seeding a boolean, not to necessarily budget my upload bandwidth across tags.

* So when I add a torrent, seeding is limited by default. (Not sure if we can disable seeding, so I just set the limit to `1B/s` bandwidth for seeding to minimize it.)
* Once I've added a tag, then it has `Unlimited` bandwidth for seeding. (I only tag torrents I intend to keep and seed.)

Whereas right now, I can only limit seeding for all torrents, so I'd have to impede all seeding to avoid seeding one torrent.

Welcome to call me out if there's a better way to achieve this!

**Reply by trim21** [reply]
I would suggest you write a script to do this
